canary/generator/transformators/transformers_categorical.py

```diff
@@ -0,0 +1,94 @@
+from abc import ABC
+from copy import deepcopy
+
+import numpy as np
+
+from .base import Transformer
+
+
+class TransformerCategorical(Transformer, ABC):
+    def __init__(self):
+        super(TransformerCategorical, self).__init__(['categorical', 'enumerated',
+                                                      'boolean', 'flag'])
+
+
+class AddToSomeBucketTransformer(TransformerCategorical):
+    def __init__(self, percentage=None, bucket_number=None, change_date=None):
+        """anomaly, works on buckets!!!"""
```

Comment:
I think I'd love a better docstring here :) Would you mind documenting the parameters as well? See [here](https://searchfox.org/mozilla-central/rev/3fdc491e118c5cdfbaf6e2d52f3466d2b27ad1de/toolkit/components/telemetry/gen_event_data.py#116-120) for an example.